dev-portal/src/services/accounts.js

```diff
@@ -0,0 +1,49 @@
+import _ from 'lodash'
+
+import { resolveAfter } from 'utils/test-utils'
+
+const now = Date.now()
+const numMockAccounts = 157 // should be prime
+const mockData = Array.from({ length: numMockAccounts }).map(
```

Comment:
Appears to be a dupe of `dev-portal/src/pages/Admin/Accounts/__tests__/RegisteredAccounts.jsx`?

Reply:
Oh, I see.

Reply:
This mock implementation is just here until the backing lambda functionality is implemented.